### PR TITLE
Preparation environment

### PR DESCRIPTION
**Pull request by jihuayu**
- Preparation environment.
- make readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,48 @@
...
+-   mofish list
+-   mofish help
+
+当然，以上功能一个都没实现。
+
+## 贡献插件
+
+0.  fork本仓库。
+1.  在ports文件中新建一个以插件名字命名的文件夹。
+2.  按照样例写好配置文档。
```
Comment (jihuayu) [inline README.md]: Codacy Issue found: [[ordered-list-marker-value] Marker should be `4`, was `3`](https://app.codacy.com/gh/mofishes/mofish/pullRequest?prid=4887991)

**Comment by jihuayu** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/mofishes/mofish/pullRequest?prid=4887991&bid=16191166)

File: README.md [hunk 2]
```diff
@@ -1,3 +1,48 @@
...
+-   mofish install <port>
+-   mofish remove <port>
+-   mofish list
+-   mofish help
+
+当然，以上功能一个都没实现。
+
+## 贡献插件
+
+0.  fork本仓库。
```
Comment (jihuayu) [inline README.md]: Codacy Issue found: [[ordered-list-marker-value] Marker should be `2`, was `1`](https://app.codacy.com/gh/mofishes/mofish/pullRequest?prid=4887991)

File: README.md [hunk 3]
```diff
@@ -1,3 +1,48 @@
...
+1.  在ports文件中新建一个以插件名字命名的文件夹。
```
Comment (jihuayu) [inline README.md]: Codacy Issue found: [[ordered-list-marker-value] Marker should be `3`, was `2`](https://app.codacy.com/gh/mofishes/mofish/pullRequest?prid=4887991)

File: README.md [hunk 4]
```diff
@@ -1,3 +1,48 @@
...
+-   src/：代码主目录。
+-   main.cpp：cli主函数，只写cli逻辑。
+
+## 安装方法
+
+-   使用mofish_install.（WIP）
+-   clone本项目自行编译.
+-   在release中下载特定版本。
+
+## 使用方法
```
Comment (zhmushan) [inline README.md]: 我以为是做類似https://github.com/CLIUtils/CLI11

Comment (jihuayu) [reply README.md]: 有两部分
- 一部分是一个用来构建命令行的库
- 一部分是用这个库构建的包管理器，用来管理用这个库构建的程序。

当然我们之前只想做第一部分，第二部分先TODO吧。

**Pull request merged**